test/layer.spec.ts

```diff
@@ -113,7 +113,21 @@ describe("applyLayers", () => {
       code: lambda.Code.fromInline("test"),
       handler: "hello.handler",
     });
-    const errors = applyLayers(stack, stack.region, [hello1, hello2], PYTHON_LAYER_VERSION, NODE_LAYER_VERSION);
+
+    const hello3 = new lambda.Function(stack, "HelloHandler3", {
+      runtime: lambda.Runtime.NODEJS_14_X,
+      code: lambda.Code.fromAsset("test/lambda"),
```

Comment:
You mentioned that the Node 14 runtime doesn't support inline code -- can the other test Lambdas in this file be updated to use this new style for consistency, or do we need two different ways in the same file?

Reply:
Good point, I've replaced the other calls with `fromAsset` in the layer.spec.ts file in my latest commit